src/libraries/System.Security.Cryptography/tests/PemEncodingFindTests.cs

```diff
@@ -467,5 +473,57 @@ protected override void AssertNoPemFound(ReadOnlySpan<char> input)
             bool found = PemEncoding.TryFind(input, out _);
             Assert.False(found, "Found PEM when not expected");
         }
+
+        protected override ReadOnlySpan<char> Create(string pem) => pem;
+    }
+
+    public class PemEncodingFindUtf8ThrowingTests : PemEncodingFindTests<byte>
+    {
+        [Fact]
+        public void FindUtf8_InvalidUtf8_OutsideOfEncapBoundary_Ignored()
```

Comment:
This succeeding test case doesn't feel like it belongs in a class named "ThrowingTests"

Reply:
The fixture name is weird because, well, that was a while ago. It means that the method under test is not a "Try" method. I can rename it if you would like.

Reply:
Oh, for Span-writing-int-returning?  Eh, good enough.